chart/identity-manager-sts/templates/deployment.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
+kind: Deployment
+metadata:
+  annotations:
```

Comment:
```suggestion
```

Reply:
Left this in, but added a key for extra annotations.